pkg/distributor/distributor.go

```diff
@@ -486,12 +516,12 @@ func (d *Distributor) createShard(streamshardCfg *shardstreams.Config, stream lo
 	}, true
 }
 
-func (d *Distributor) boundsFor(stream logproto.Stream, totalShards, shardNumber int, loggingEnabled bool) (int, int, bool) {
+func (d *Distributor) boundsFor(stream logproto.Stream, totalShards, spot int, loggingEnabled bool) (int, int, bool) {
```

Comment:
What do you think about renaming `spot` to `index` or `idx`? I know the type is going to be awkward but it took me awhile to understand what was meant by `spot`?

Reply:
idx/index feels much better, agreed